tools/reviewer/ReviewerConfigTest.java

```diff
@@ -53,5 +57,20 @@ public void setup() {
   public void protoFileTest() throws Exception {
     fileUtils.readPrototxt("reviewer_config.prototxt", ReviewerConfig.newBuilder());
   }
+
+  @Test
+  public void TotalCrystalCorrectTest() throws IOException {
```

Comment:
Method names should start with lowercase, so:
s/TotalCrystalCorrectTest/totalCrystalCorrectTest